auth-web/src/components/auth/common/AccountBusinessType.vue

```diff
@@ -176,151 +181,241 @@
 </template>
 
 <script lang="ts">
-import { Action, State } from 'pinia-class'
-import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
-import { OrgBusinessType, Organization } from '@/models/Organization'
-import { Account } from '@/util/constants'
-import { Code } from '@/models/Code'
+import { AccessType, Account, AccountType, SessionStorageKeys } from '@/util/constants'
+import { computed, defineComponent, nextTick, onMounted, ref } from '@vue/composition-api'
+import ConfigHelper from '@/util/config-helper'
+import { OrgBusinessType } from '@/models/Organization'
 import OrgNameAutoComplete from '@/views/auth/OrgNameAutoComplete.vue'
 import { useCodesStore } from '@/stores/codes'
 import { useOrgStore } from '@/stores/org'
 
-@Component({
+export default defineComponent({
+  name: 'AccountBusinessType',
   components: {
     OrgNameAutoComplete
-  }
-})
-export default class AccountBusinessType extends Vue {
-  @Prop({ default: false }) govmAccount: boolean
-  @Prop({ default: null }) errorMessage: string
-  @Prop({ default: false }) saving: boolean
-  @Prop({ default: null }) bcolDuplicateNameErrorMessage: string
-  @Prop({ default: false }) premiumLinkedAccount: boolean
-  @Prop({ default: false }) orgNameReadOnly: boolean
-  @Prop({ default: false }) isEditAccount: boolean // hide some details for update account
-
-  @State(useOrgStore) public currentOrganization!: Organization
+  },
+  props: {
+    govmAccount: {
+      type: Boolean,
+      default: false
+    },
+    errorMessage: {
+      type: String,
+      default: null
+    },
+    saving: {
+      type: Boolean,
+      default: false
+    },
+    bcolDuplicateNameErrorMessage: {
+      type: String,
+      default: null
+    },
+    premiumLinkedAccount: {
+      type: Boolean,
+      default: false
+    },
+    orgNameReadOnly: {
+      type: Boolean,
+      default: false
+    },
+    isEditAccount: {
+      type: Boolean,
+      default: false
+    }
+  },
+  setup (props, { emit }) {
+    const codesStore = useCodesStore()
+    const orgStore = useOrgStore()
 
-  @Action(useCodesStore) private readonly getBusinessSizeCodes!: () => Promise<Code[]>
-  @Action(useCodesStore) private readonly getBusinessTypeCodes!: () => Promise<Code[]>
-  @State(useCodesStore) private readonly businessSizeCodes!: Code[]
-  @State(useCodesStore) private readonly businessTypeCodes!: Code[]
+    const currentOrganization = computed(() => orgStore.currentOrganization)
+    const businessSizeCodes = computed(() => codesStore.businessSizeCodes)
+    const businessTypeCodes = computed(() => codesStore.businessTypeCodes)
```

Comment:
I asked Rodrigo in his PR, is it possible to just return the orgStore instead of wrapping it in a computed? Just curious

Reply:
I guess the computed makes it readonly

Reply:
I think if directly use it, will not update reactively when currentOrganization changes?

Reply:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/575ff9a4-f717-40b0-9f9d-49482f01919e)
https://pinia.vuejs.org/core-concepts/